lib/vault.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+const { exec } = require('child_process');
+const fs = require('fs');
+
+const service = process.env.SERVICE_SYSTEM_CODE;
+const region = (process.env.REGION || 'LOCAL').toLowerCase();
+const vaultPath = `secret/teams/origami/${service}/${region}`;
+const envFile = '.env';
+
+if (!service) {
+    console.warn(
+        'Could not load environment variables from Vault. Service name is undefined.',
+        'Set the environment variable "SERVICE_SYSTEM_CODE" in your Makefile.'
```

Comment:
What is a service name? Is it the same as a system code?

Reply:
Good spot.